class/defaults.yml

```diff
@@ -134,3 +158,10 @@ parameters:
           registry: quay.io
         master:
           labels: ${keycloak:labels}
+        volumePermissions:
+          enabled: ${keycloak:database:tls:enabled}
+        tls:
+          enabled: ${keycloak:database:tls:enabled}
+          certificatesSecret: ${keycloak:database:tls:certSecretName}
+          certFilename: tls.crt
+          certKeyFilename: tls.key
```

Comment:
Looking at this, it seems like the certificate secret must exist when tls is enabled, regardless of verification mode?

Reply:
Yes, as this configures the certs to use for the postgresql server. Could be selfsigned, LE or bought certs etc.

Reply:
Ah sorry, I misinterpreted the location of this change, I thought this was some additional config for Keycloak.